### DISCUSSION TITLE
Apply horizontal flip?

### DISCUSSION THREAD

**Discussion by Jonseed** [d8ahazard/sd_dreambooth_extension#126] [Q&A] [answered]
If your dataset already has horizontally flipped images, such as generated by the Preprocess Images function under Train, then is it unnecessary to select the "Apply horizontal flip" option in Dreambooth? Does that just duplicate the flips?

**Comment by d8ahazard** [comment, marked as answer]
That is correct. What the hflip option in this case does is randomly flips input images. I have no solid data to back this theory up, but I suspect this will work better than creating flipped images, as there's less chance of the model "over-training" on specific backgrounds and stuff like that.

**Comment by toyxyz** [comment]
If the character you want to train is different from left to right(Eye color, hairstyle, clothes, etc.), it is better to turn it off. In some cases, an image inverted left and right is generated.

**Comment by leoftm** [comment]
This is off topic, but a completely doubled watered-down horizontal flip training data would learn a symmetrical hairstyle, parted in the middle, from a left-tilted and right-tilted hairdo. Perhaps random flips are less pronounced, but not really recommended, maybe.
